### PR TITLE
Update active-active.html.md

### PR DESCRIPTION
**Pull request by sudomateo**
## Labels

- [x] inaccuracy
- [ ] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/enterprise/admin/active-active.html.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Note that `tfe-admin` is an alias for `replicated admin`, and can be used interc
 tfe-admin support-bundle
 ```
 
-This command will generate a support bundle for all nodes. The support bundles will be uploaded to the same object store bucket that is used to store Terraform statefiles. The support bundles for a specific run of the admin command will all be uploaded to a directory with the same JobID (a timestamp in [RFC3339](https://tools.ietf.org/html/rfc3339) format). If you are sending a support bundle to HashiCorp Support, package up the associated bundles and send all in order to ensure all needed information is made available.
+This command will generate a support bundle for all nodes. The support bundles will be uploaded to the same object store bucket that is used to store Terraform statefiles. The support bundles for a specific run of the admin command will all be uploaded to a directory with the same JobID (a timestamp in [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) format). If you are sending a support bundle to HashiCorp Support, package up the associated bundles and send all in order to ensure all needed information is made available.
```
Comment (laurapacilio) [inline content/source/docs/enterprise/admin/active-active.html.md:24]: It's cleaner without "will" and also this matches the construction we have below :-) 

```suggestion
This command generates a support bundle for all nodes. The support bundles will be uploaded to the same object store bucket that is used to store Terraform state files. The support bundles for a specific run of the admin command will all be uploaded to a directory with the same JobID (a timestamp in [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) format). If you are sending a support bundle to HashiCorp Support, package and send all associated bundles to ensure that we have all the necessary information.
```

File: content/source/docs/enterprise/admin/active-active.html.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ This command will quiesce the current node and remove it from service. It will a
...
+This command allows users to make ad hoc/realtime application changes, such as `capacity_concurrency` via the CLI. Both an allowable `<KEY>` (setting name) and `<VALUE>` (new setting value) must be provided. A complete list of the current `app-config` settings can be found by running `replicatedctl app-config export`. After making configuration changes, the Terraform Enterprise application must be restarted **on each node instance** for the configuration changes to take effect. To restart Terraform Enterprise, first stop the application by executing `replicatedctl app stop`. Next, execute `replicatedctl app status` to confirm the application is stopped. Finally, execute `replicatedctl app start` to start the application.
```
Comment (laurapacilio) [inline content/source/docs/enterprise/admin/active-active.html.md:53]: Some updates for active voice and also to help the text flow a bit better. For example, I love that you included steps to restart TFE, and I wanted to help the users parse them more quickly by using a numbered list.
```suggestion
This command allows you to use the CLI to make real-time application changes, such as `capacity_concurrency`. You must provide both an allowable `<KEY>` (setting name) and `<VALUE>` (new setting value). Run `replicatedctl app-config export` for a complete list of the current `app-config` settings. 

For the configuration changes to take effect, you must restart the Terraform Enterprise application **on each node instance**. To restart Terraform Enterprise:
1. Run `replicatedctl app stop` to stop the application. 
2. Run `replicatedctl app status` to confirm the application is stopped. 
3. Run `replicatedctl app start` to start the application.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Thank you for adding! I'm not reviewing for content accuracy, but I did review for style and flow. I added a couple of suggestions for your review - please make sure that my edits didn't unintentionally change the meaning!

**Comment by sudomateo** [comment]
Thank you for the review! Implemented!

**Review by laurapacilio — APPROVED** [review]

**Comment by laurapacilio** [comment]
Nice - thank you! :-) Feel free to merge whenever!

**Pull request merged**